dapps/dapps.json

```diff
@@ -0,0 +1,27 @@
+[
+  {
+    "category": "DeFi",
```

Comment:
What we should do if one DApp will contain in more than one category? May it would be better specify category as parameter for each DApp?